File: guanabara-java/src/com/danillo/algoritmos/lista03/Exercicio35.java
package com.danillo.algoritmos.lista03;

import java.util.Scanner;

public class Exercicio35 {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);

        System.out.println("Insira o tipo de carro a ser alugado: 1- Carro popular 2-Carro de luxo");
        int tipoCarro = entrada.nextInt();
        System.out.println("Insira a quantidade de dias que o carro foi alugado: ");
        int diasAluguel = entrada.nextInt();
        System.out.println("Insira a quantidade de km que foram percorridos: ");
        int kmPercorridos = entrada.nextInt();

        int valorTempoAluguel;
        float valorDistanciaPercorrida;

        if(tipoCarro == 1){
            valorTempoAluguel = diasAluguel * 90;

            if(kmPercorridos <= 100){
                valorDistanciaPercorrida = kmPercorridos * 0.20f;
            }else{
                valorDistanciaPercorrida = kmPercorridos * 0.10f;
            }
        }else{
            valorTempoAluguel = diasAluguel * 150;

            if(kmPercorridos <= 200){
                valorDistanciaPercorrida = kmPercorridos * 0.30f;
            }else{
                valorDistanciaPercorrida = kmPercorridos * 0.25f;
            }
        }

        float valorAluguel = valorTempoAluguel + valorDistanciaPercorrida;
        System.out.println("Valor do aluguel: R$ " +valorAluguel);
    }
}

File: guanabara-python/src/lista03/exercicio35.py
tipo_carro = int(input("Insira o tipo do carro: 1 - Carro popular. 2 - Carro esportivo. "))
quantidade_dias = int(input("Insira a quantidade de dias em que o carro será alugado: "))
quantidade_km_percorridos = int(input("Insira a distância a ser percorrida (em km): "))

if tipo_carro == 1:
    if quantidade_km_percorridos <= 100:
        preco_km = quantidade_km_percorridos * 0.20
        preco_dia = quantidade_dias * 90
        preco_total = preco_km + preco_dia
    else:
        preco_km = quantidade_km_percorridos * 0.10
        preco_dia = quantidade_dias * 90
        preco_total = preco_km + preco_dia
elif tipo_carro == 2:
    if quantidade_km_percorridos <= 200:
        preco_km = quantidade_km_percorridos * 0.30
        preco_dia = quantidade_dias * 150
        preco_total = preco_km + preco_dia
    else:
        preco_km = quantidade_km_percorridos * 0.25
        preco_dia = quantidade_dias * 150
        preco_total = preco_km + preco_dia
else:
    print("O tipo de carro inserido não existe.")

print(f"Preço da viagem: R$ {preco_total}")
